### PR TITLE
setup your ide

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [x] Ehancement

- Related issues

### CODE REVIEW

File: docs/en/guides/How-to-build.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ a oap image with name `bar/oap:foo`, run the following command
 > HUB=bar TAG=foo make docker.oap
 
 ## Setup your IntelliJ IDEA
+**NOTICE**: If you clone the codes from Github, please make sure that you had finished step 1 to 7 in section **Build from GitHub**.
```
Comment (kezhenxu94) [inline docs/en/guides/How-to-build.md]:
```suggestion
**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **Build from GitHub**.
```

File: docs/en/guides/How-to-build.md [hunk 2]
```diff
@@ -77,6 +79,8 @@ a oap image with name `bar/oap:foo`, run the following command
     * `oal` folder in **oap-server/generated-analysis/target/generated-sources**
     
 ## Setup your Eclipse IDE
+**NOTICE**: If you clone the codes from Github, please make sure that you had finished step 1 to 7 in section **Build from GitHub**.
```
Comment (kezhenxu94) [inline docs/en/guides/How-to-build.md]:
```suggestion
**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **Build from GitHub**.
```

File: docs/en/guides/How-to-build.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ a oap image with name `bar/oap:foo`, run the following command
 > HUB=bar TAG=foo make docker.oap
 
 ## Setup your IntelliJ IDEA
+**NOTICE**: If you clone the codes from Github, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-apache-source-code-release)**.
```
Comment (wu-sheng) [inline docs/en/guides/How-to-build.md]: Also, text is not build from GitHub, you are buildin from source codes.

Comment (dmsolr) [reply docs/en/guides/How-to-build.md]: yes. i will fix later

**Comment by wu-sheng** [comment]
I think you misunderstand my point. We have source code in two places, one is GitHub, the other Apache source release. They are different, Apache soure release doesn't need git submodule thing, because we did that in source release procedure.

**Comment by dmsolr** [comment]
> I think you misunderstand my point. We have source code in two places, one is GitHub, the other Apache source release. They are different, Apache soure release doesn't need git submodule thing, because we did that in source release procedure.

Hi, @wu-sheng . I got it. So here's the case with code clone from GitHub. 
> **If you clone the codes from Github**, please make sure that you had finished step 1 to 7 in section Build from GitHub

If download Apache soure release, we don't need that.

File: docs/en/guides/How-to-build.md [hunk 4]
```diff
@@ -65,6 +65,8 @@ a oap image with name `bar/oap:foo`, run the following command
 > HUB=bar TAG=foo make docker.oap
 
 ## Setup your IntelliJ IDEA
+**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-github)**.
```
Comment (kezhenxu94) [inline docs/en/guides/How-to-build.md]:
```suggestion
**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-github)**, if you download the source codes from the official website of SkyWalking, please make sure that you had followed the steps in section **[Build from Apache source code release](#build-from-apache-source-code-release)**.
```

File: docs/en/guides/How-to-build.md [hunk 5]
```diff
@@ -77,6 +79,8 @@ a oap image with name `bar/oap:foo`, run the following command
     * `oal` folder in **oap-server/generated-analysis/target/generated-sources**
     
 ## Setup your Eclipse IDE
+**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-github)**.
```
Comment (kezhenxu94) [inline docs/en/guides/How-to-build.md]:
```suggestion
**NOTICE**: If you clone the codes from GitHub, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-github)**, if you download the source codes from the official website of SkyWalking, please make sure that you had followed the steps in section **[Build from Apache source code release](#build-from-apache-source-code-release)**.
```

**Comment by kezhenxu94** [comment]
> If download Apache soure release, we don't need that.

You still need that to generate some source codes from `.proto` files

**Comment by dmsolr** [comment]
> > If download Apache soure release, we don't need that.
> 
> @dmsolr You still need that to generate some source codes from `.proto` files

Yes. There are `Run ./mvnw compile -Dmaven.test.skip=true to compile project and generate source codes. Because we use gRPC and protobuf` in the steps. 
But it is ok that we say it twice.

**Pull request merged**